djconnectwise/models.py

```diff
@@ -446,6 +446,42 @@ def __str__(self):
         return self.name
 
 
+class CompanyTeam(models.Model):
+    company = models.ForeignKey('Company', blank=True,
+                                null=True, on_delete=models.CASCADE)
+    team_role = models.ForeignKey('CompanyTeamRole',
+                                  blank=True, null=True,
+                                  on_delete=models.CASCADE)
+    location = models.ForeignKey('Location', blank=True,
+                                 null=True, on_delete=models.CASCADE)
+    contact = models.ForeignKey('Contact', blank=True,
+                                null=True, on_delete=models.CASCADE)
+    member = models.ForeignKey('Member', blank=True,
+                               null=True, on_delete=models.CASCADE)
+    accountManagerFlag = models.BooleanField()
```

Comment:
Lets stick to snake case. Can you change this to `account_manager_flag` instead? And any other cases where camel case is used on the new models?